src/overlays/actors/ovl_En_Trt/z_en_trt.c

```diff
@@ -22,176 +103,1714 @@ const ActorInit En_Trt_InitVars = {
+        gDPPipeSync(OVERLAY_DISP++);
+        gDPSetTile(OVERLAY_DISP++, G_IM_FMT_IA, G_IM_SIZ_8b, 2, 0x0000, G_TX_RENDERTILE, 0, G_TX_NOMIRROR | G_TX_WRAP,
+                   G_TX_NOMASK, G_TX_NOLOD, G_TX_NOMIRROR | G_TX_WRAP, 4, G_TX_NOLOD);
+        gDPSetTileSize(OVERLAY_DISP++, G_TX_RENDERTILE, 0, 0, 15 * 4, 23 * 4);
+        if (drawStickRightPrompt) {
+            EnTrt_DrawTextRec(globalCtx, this->stickLeftPrompt.arrowColorR, this->stickLeftPrompt.arrowColorG,
+                          this->stickLeftPrompt.arrowColorB, this->stickLeftPrompt.arrowColorA,
+                          this->stickLeftPrompt.arrowTexX, this->stickLeftPrompt.arrowTexY, this->stickLeftPrompt.texZ,
+                          0, 0, -1.0f, 1.0f);
+        }
+        if (drawStickLeftPrompt) {
+            EnTrt_DrawTextRec(globalCtx, this->stickRightPrompt.arrowColorR, this->stickRightPrompt.arrowColorG,
+                          this->stickRightPrompt.arrowColorB, this->stickRightPrompt.arrowColorA,
+                          this->stickRightPrompt.arrowTexX, this->stickRightPrompt.arrowTexY,
+                          this->stickRightPrompt.texZ, 0, 0, 1.0f, 1.0f);
+        }
+        gDPSetTextureImage(OVERLAY_DISP++, G_IM_FMT_IA, G_IM_SIZ_16b, 1, &D_0401F7C0);
+        gDPSetTile(OVERLAY_DISP++, G_IM_FMT_IA, G_IM_SIZ_16b, 0, 0x0000, G_TX_LOADTILE, 0, G_TX_NOMIRROR | G_TX_WRAP,
+                   G_TX_NOMASK, G_TX_NOLOD, G_TX_NOMIRROR | G_TX_WRAP, 4, G_TX_NOLOD);
+        gDPLoadSync(OVERLAY_DISP++);
+        gDPLoadBlock(OVERLAY_DISP++, G_TX_LOADTILE, 0, 0, 127, 1024);
+        gDPPipeSync(OVERLAY_DISP++);
+        gDPSetTile(OVERLAY_DISP++, G_IM_FMT_IA, G_IM_SIZ_8b, 2, 0x0000, G_TX_RENDERTILE, 0, G_TX_NOMIRROR | G_TX_WRAP,
+                   G_TX_NOMASK, G_TX_NOLOD, G_TX_NOMIRROR | G_TX_WRAP, 4, G_TX_NOLOD);
+        gDPSetTileSize(OVERLAY_DISP++, G_TX_RENDERTILE, 0, 0, 15 * 4, 15 * 4);
+        if (drawStickRightPrompt) {
+            EnTrt_DrawTextRec(globalCtx, this->stickLeftPrompt.stickColorR, this->stickLeftPrompt.stickColorG,
+                          this->stickLeftPrompt.stickColorB, this->stickLeftPrompt.stickColorA,
+                          this->stickLeftPrompt.stickTexX, this->stickLeftPrompt.stickTexY, this->stickLeftPrompt.texZ,
+                          0, 0, -1.0f, 1.0f);
+        }
+        if (drawStickLeftPrompt) {
+            EnTrt_DrawTextRec(globalCtx, this->stickRightPrompt.stickColorR, this->stickRightPrompt.stickColorG,
+                          this->stickRightPrompt.stickColorB, this->stickRightPrompt.stickColorA,
+                          this->stickRightPrompt.stickTexX, this->stickRightPrompt.stickTexY,
+                          this->stickRightPrompt.texZ, 0, 0, 1.0f, 1.0f);
+        }
+    }
+    CLOSE_DISPS(globalCtx->state.gfxCtx);
+}
+
+void EnTrt_Init(Actor* thisx, GlobalContext* globalCtx) {
+    static InitChainEntry sInitChain[] = {
+        ICHAIN_F32(targetArrowOffset, 500, ICHAIN_STOP),
+    };
+
```

Comment:
We usually keep the InitChain out of the Init function.

Reply:
So I can't bring the initChain all the way up to the top, as there is static data above it that requires it to be inside the function to match (`sSelectedItemPosition` in `EnTrt_PositionSelectedItem`). However I will take the initChain out of the function and put it just above it.